### PR TITLE
Adding unit Test for HttpAdapter + Code Cleaning 

### PR DESCRIPTION
**Pull request by mestizoLopez**
# Description

Creating Unit Tests for DaprClientHttpAdapter. I add a new dependency for testing junit -jupiter version 5
also i removed an unnecessary test and unused imports.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[22]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ X ] Code compiles correctly
* [ X ] Created/updated tests
* [ X ] Extended the documentation

### CODE REVIEW

File: sdk/src/test/java/io/dapr/client/DaprClientHttpAdapterTest.java [hunk 1]
```diff
@@ -0,0 +1,671 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+package io.dapr.client;
```
Comment (LMWF) [inline sdk/src/test/java/io/dapr/client/DaprClientHttpAdapterTest.java]: General comment not on this file: if you are using intellij can you please `Code` menu -> Reformat Code.  There is odd formatting.

Comment (youngbupark) [reply sdk/src/test/java/io/dapr/client/DaprClientHttpAdapterTest.java]: +1 there are too many return spaces between the lines.

Comment (mestizoLopez) [reply sdk/src/test/java/io/dapr/client/DaprClientHttpAdapterTest.java]: It is done. Can you check again please? Let me know if there is any other change i need to do.

Comment (artursouza) [reply sdk/src/test/java/io/dapr/client/DaprClientHttpAdapterTest.java]: We are using 2 spaces, BTW.

Comment (artursouza) [reply sdk/src/test/java/io/dapr/client/DaprClientHttpAdapterTest.java]: image

**Pull request closed without merging**